dotnet/src/Connectors/Connectors.Google/Core/Gemini/Models/GeminiRequest.cs

```diff
@@ -26,6 +26,10 @@ internal sealed class GeminiRequest
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public IList<GeminiTool>? Tools { get; set; }
 
+    [JsonPropertyName("systemInstruction")]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public GeminiContent? SystemMessages { get; set; }
```

Comment:
Property name is plural, although JSON property name is `systemInstruction` and we keep single instance of `GeminiContent`. I would probably use `SystemInstruction` name for the property as well. Then you will have `SystemInstruction.Parts` to access multiple messages.